drivers/ws2812_common.h

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
I feel like this should just be in `ws2812.h`, but currently there are two of them for AVR and ChibOS. The only difference is the latter doesn't have `ws2812_setleds_pin()`, but it probably could. Possibly that could be done in #9413.

Reply:
Or could have a single `ws2812.h` with this:
```c
#ifdef __AVR__
    void ws2812_setleds_pin(LED_TYPE *ledarray, uint16_t number_of_leds, uint8_t pin);
#endif
```

But that seems uglier to me...

Reply:
Honestly, `ws2812_setleds_pin` should probably be removed, outright, since you can't actually dynamically set the pin.

Reply:
So, remove `ws2812_setleds_pin()` from the codebase, and combine the three `ws2812*.h` files into a single `.../drivers/ws2812.h`?

Reply:
Done.